src/azure/storage/signer.rs

```diff
@@ -22,10 +22,11 @@ use crate::time::format_http_date;
 use crate::time::DateTime;
 
 /// Builder for `Signer`.
-#[derive(Default)]
+#[derive(Default, Clone)]
 pub struct Builder {
     credential: Credential,
     allow_anonymous: bool,
+    is_batch: bool,
```

Comment:
I don't think `is_batch` is a good name. How about using `omit_service_version` instead?